internal/website/content/howto/docstore/_index.md

```diff
@@ -26,49 +26,186 @@ way. To be portable, Docstore requires that the key be part of the document's
 contents. When you open a collection using one of the functions described here,
 you specify how to find the provider's primary key in the document.
 
-[`*docstore.Collection`]: https://godoc.org/gocloud.dev/docstore#Collection
-
-### Constructors versus URL openers
-
 The easiest way to open a collection is using [`docstore.OpenCollection`][] and
 a URL pointing to the topic, making sure you ["blank import"][] the driver
 package to link it in. See [Concepts: URLs][] for more details. If you need
 fine-grained control over the connection settings, you can call the constructor
 function in the driver package directly (like `mongodocstore.OpenCollection`).
-This guide will show how to use both forms for each document store provider.
 
+See the [guide below][] for usage of both forms for each supported provider.
+
+[`*docstore.Collection`]: https://godoc.org/gocloud.dev/docstore#Collection
 [`docstore.OpenCollection`]:
 https://godoc.org/gocloud.dev/docstore#OpenCollection
 ["blank import"]: https://golang.org/doc/effective_go.html#blank_import
+[Concepts: URLs]: {{< ref "/concepts/urls.md" >}}
+[guide below]: {{< ref "#services" >}}
 
-\[Concepts: URLs]: {{< ref "/concepts/urls.md" >}}
+## Using a Collection
 
-### DynamoDB {#dynamodb}
+### Representing Documents {#rep-doc}
 
-The [`awsdynamodb`](https://godoc.org/gocloud.dev/docstore/awsdynamodb) package
-supports [Amazon DynamoDB](https://aws.amazon.com/dynamodb). A Docstore
-collection corresponds to a DynamoDB table.
+We'll use a collection with documents represented by this Go struct:
 
-DynamoDB URLs provide the table, partition key field and optionally the sort key
-field for the collection.
+```go
+type Player struct {
+    Name             string
+    Score            int
+    DocstoreRevision interface{}
+}
+```
 
-{{< goexample
-"gocloud.dev/docstore/awsdynamodb.Example_openCollectionFromURL" >}}
+We recommend using structs for documents because they impose some structure on
+your data, but Docstore also accepts `map[string]interface{}` values. See
+[the `docstore` package documentation](https://godoc.org/gocloud.dev/docstore#hdr-Documents)
+for more information.
 
-Full details about acceptable URLs can be found under the API reference for
-[`awsdynamodb.URLOpener`][].
+The `DocstoreRevision` field holds information about the latest revision of the
+document. We discuss it [below]({{< ref "#docrev" >}}).
 
-#### DynamoDB Constructor {#dynamodb-ctor}
+### Actions {#actions}
 
-The [`awsdynamodb.OpenCollection`][] constructor opens a DynamoDB table as a
-Docstore collection. You must first create an [AWS session][] with the same
-region as your collection:
+In Docstore, you use actions to read, modify and write documents. You can
```

Comment:
This is because I moved representing documents out and the action summary is now next to the action list one. I think it is still useful to explain single action and action list. I reworded it a little bit.